cmd/server-main.go

```diff
@@ -73,22 +73,22 @@ EXAMPLES:
       $ minio {{.Name}} C:\MyShare
 
   4. Start minio server on 12 disks to enable erasure coded layer with 6 data and 6 parity.
-      $ minio {{.Name}} /mnt/export1/backend /mnt/export2/backend /mnt/export3/backend /mnt/export4/backend \
-          /mnt/export5/backend /mnt/export6/backend /mnt/export7/backend /mnt/export8/backend /mnt/export9/backend \
-          /mnt/export10/backend /mnt/export11/backend /mnt/export12/backend
+      $ minio {{.Name}} /mnt/export1/ /mnt/export2/ /mnt/export3/ /mnt/export4/ \
+          /mnt/export5/ /mnt/export6/ /mnt/export7/ /mnt/export8/ /mnt/export9/ \
+          /mnt/export10/ /mnt/export11/ /mnt/export12/
 
   5. Start minio server on 12 disks while ignoring two disks for initialization.
-      $ minio {{.Name}} --ignore-disks=/mnt/export1/backend,/mnt/export2/backend /mnt/export1/backend \
-          /mnt/export2/backend /mnt/export3/backend /mnt/export4/backend /mnt/export5/backend /mnt/export6/backend \
-          /mnt/export7/backend /mnt/export8/backend /mnt/export9/backend /mnt/export10/backend /mnt/export11/backend \
-          /mnt/export12/backend
+      $ minio {{.Name}} --ignore-disks=/mnt/export1/ /mnt/export1/ /mnt/export2/ \
```

Comment:
@krisis, it has to be like that, otherwise:
```
$ minio server --ignore-disks=/mnt/p8/xl/backend1/ /mnt/p8/xl/backend{2..12}
FATA[0001] Invalid disk arguments for server.            cause=Total number of disks should be multiples of '2'
```